examples/kotlin-diktat/sarif-actual/save-warnings-actual.sarif

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
seems, that `execute warn-plugin with actual warnings from SARIF` test didn't worked at all, since there wasn't any sarif file for this test